### ISSUE TITLE
Notification content is blank for text strings matching "<*"

### ISSUE THREAD

**Issue by sashaweiss** [klinker-apps/messenger-issues#635]
- [x] I have [searched](https://github.com/klinker-apps/messenger-issues/issues), by keyword, for my issue/request to verify that there aren't any open or closed issues around this request.
- [x] If this is an issue, and not a request, I have verified that this is something that is reproducible

**Device with issue:** Google Pixel 2 running Android 8.1.0
**Primary device:** same as above
**App version:** 2.9.4.1995
**Steps to reproduce:**

1. Receive a text containing the character "<", optionally immediately followed by other characters (e.g. "<3", "<12345", "<asdf").
2. Open the Notifications drawer, and look at the Pulse notification.
3. Notice that the string beginning with the "<" won't appear - for example, a text containing just "<asdf" will appear to be a blank text in the notification.

**Additional information:**
If there's a space directly after the "<", the character appears. Collected a couple [screenshots](https://imgur.com/a/mDKMo) from trying it out - one is a screenshot of the notification, and the other is a screenshot of the actual text conversation (the actual text received).

Not a huge priority - but thanks for taking a look!!

**Comment by klinker24** [comment]
Yeah, I suppose it would be. Notifications support spannables for the strings, which contain HTML content. I guess Android is picking that up as HTML content. I will have to figure something out around this.

**Comment by sashaweiss** [comment]
That makes sense - thanks for the quick reply!

**Comment by sashaweiss** [comment]
Awesome, thanks!
